torch/csrc/jit/constants.cpp

```diff
@@ -47,7 +49,7 @@ Value* insertConstant(
     n->ts_(
         attr::value,
         fmap(val.toTensorList()->elements(), [](const at::Tensor& t) {
-          return autograd::Variable(t).data();
+          return const_cast<at::Tensor&>(t).set_requires_grad(false);
```

Comment:
This should be an assertion that `t` does not require grad. We should not be passing grad recording tensors here.